app/_components/GuessDistanceModal.tsx

```diff
@@ -123,33 +124,18 @@ const GuessDistanceModal = ({
                 onChange={(e) => setHint(e.target.value)}
                 className="border p-2 mt-2 w-full"
               />
-              <Input
-                type="text"
-                value={hint}
-                onChange={(e) => setHint(e.target.value)}
-              />
               <div className="flex flex-col gap-4 mt-4 justify-end">
-                <Button
-                  ref={drawerRef}
-                  onClick={() => {
-                    handleSubmitClick;
-                    // setGuessPoiPosition(null);
-                    setGuessedPin(null);
-                  }}
-                  className="w-full"
-                >
-                  Submit Hint
-                </Button>
```

Comment:
We need 2 buttons, one for submit and one for cancel in case users don't want to swipe or tab the black transparent screen. also submit should have a variant default while cancel will use `variant={'link'}` to separate the these buttons